# https://git.exherbo.org/media.git/commit/?id=b7a022928103184b7528d0e40ad0a4edcfbbd7a4

From 8a9d11e96b771422247c458ae6b07f3a13848f3a Mon Sep 17 00:00:00 2001
From: Timo Gurr <timo.gurr@gmail.com>
Date: Sat, 15 Jan 2022 20:44:53 +0100
Subject: [PATCH] FFmpeg deprecations fixes for FFmpeg >= 5

avcodec.h stopped including bsf.h per FFmpeg commit
57b5ec6ba7df [1]. Fixes compilation error against
FFmpeg later than the mentioned commit.
[1] https://github.com/FFmpeg/FFmpeg/commit/57b5ec6ba7df442caebc401c4a7ef3ebc066b519
---
 libhb/decavcodec.c  | 1 +
 libhb/muxavformat.c | 1 +
 2 files changed, 2 insertions(+)

diff --git a/libhb/decavcodec.c b/libhb/decavcodec.c
index 372f86f82cf..e7a63289c0f 100644
--- a/libhb/decavcodec.c
+++ b/libhb/decavcodec.c
@@ -41,6 +41,7 @@
 #include "handbrake/handbrake.h"
 #include "handbrake/hbffmpeg.h"
 #include "handbrake/hbavfilter.h"
+#include "libavcodec/bsf.h"
 #include "libavfilter/avfilter.h"
 #include "libavfilter/buffersrc.h"
 #include "libavfilter/buffersink.h"
diff --git a/libhb/muxavformat.c b/libhb/muxavformat.c
index 232edd0942a..ad8e2a1d8fa 100644
--- a/libhb/muxavformat.c
+++ b/libhb/muxavformat.c
@@ -8,6 +8,7 @@
  */
 
 #include <ogg/ogg.h>
+#include "libavcodec/bsf.h"
 #include "libavformat/avformat.h"
 #include "libavutil/avstring.h"
 #include "libavutil/intreadwrite.h"
